public/scripts/main.js

```diff
@@ -127,12 +129,23 @@ const gameHandler = (function gameIIFE() {
     })
   }
 
+  function displayRole(role) {
+    document.querySelector('#image').src = role.image
+    document.querySelector('#name').textContent = role.name
+    document.querySelector('#description').textContent = role.description
+  }
+
   // Transitions
   function onEnter() {
     socket.on('message', appendIncomingMessage)
     input.addEventListener('input', validateInput)
     input.addEventListener('keyup', enterKeyPressed(sendMessage))
     game.hidden = false
+    const STARTING_REQUIRED_WORD_COUNT = 10
```

Comment:
Do you think this would make more sense to declare at the top of the handler itself, rather than within the `onEnter` function? I think it might be needed during the sentence validation step of checking that you have a certain percentage of the words present before submitting. That way we can set it up as some ratio rather than hard coding (so "you need to use 25% of the words" rather than "you need to use 4 of the words")